src/core.rs

```diff
@@ -211,16 +233,18 @@ pub struct Core {
     our_connection_info_map: LruCache<PublicId, OurConnectionInfo>,
     their_connection_info_map: LruCache<PublicId, TheirConnectionInfo>,
     get_request_count: usize,
```

Comment:
That was just a quick and dirty change to get those stats out of the way, but yeah, anything that is just statistics for debugging purposes should probably go there too.